website/server/models/subscriptionPlan.js

```diff
@@ -16,6 +16,9 @@ export let schema = new mongoose.Schema({
   gemsBought: {type: Number, default: 0},
   mysteryItems: {type: Array, default: () => []},
   lastBillingDate: Date, // Used only for Amazon Payments to keep track of billing date
+  additionalData: mongoose.Schema.Types.Mixed,
```

Comment:
Also at line 10 the comment should include Google among the valid payment methods